src/ResourceManager/MachineLearningCompute/ChangeLog.md

```diff
@@ -18,10 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Add IncludeAllResources parameter to Remove-AzureRmMlOpCluster cmdlet
+    - Using this switch parameter will remove all resources that were created with the cluster originally
+
+## Version 0.3.1
```

Comment:
Don't modify this - just add your snippet after the snippets about ResourceGroup and Location completers.  We will automatically modify this file during release.

Reply:
That makes sense. I looked at the version number in the .psd1 file and it shows 0.3.1 which is below the current release so that's why I thought the version numbers are off. But maybe I'm misunderstanding?

Reply:
"Current release" actually means the upcoming release: it will be displayed as the current release notes in the next release.